reporting/api/src/main/java/org/jboss/windup/reporting/config/Classification.java

```diff
@@ -181,6 +179,26 @@ public String toString()
         return result.toString();
     }
 
+    public List<Link> getLinks()
+    {
+        return links;
+    }
+
+    public RegexParameterizedPatternParser getClassificationPattern()
```

Comment:
I fixed what I could, some are not easily fixable. For example Link.to() needs to return Link, because it is sent to Hint that needs to access .get*() methods. The same goes for Artifact and Version.